pyproject.toml

```diff
@@ -18,3 +18,19 @@ convention = "google"
 
 [tool.ruff.mccabe]
 max-complexity = 10
+
+[tool.mypy]
+python_version = "3.9"
```

Comment:
Shouldn't we go for a newer version of python here?

Reply:
Pyresample hasn't dropped Python 3.9 yet so I think it needs to (or should) match what the package's compatibility is specified as (3.9+).